### PR TITLE
Add flag to skip commit UUID validation

### PR DESCRIPTION
**Pull request by hjrocha**
Based on task CY-4243

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,6 +211,7 @@ Found [Clone] 7 duplicated lines with 10 tokens:
     * `--output` - Send the output results to a file
     * `--format` [default: text] - Change the output format (e.g. json)
     * `--commit-uuid` [default: latest commit of current git branch] - Set the commit UUID that will receive the results on Codacy
+    * `--skip-commit-uuid-validation` | -s  Skip commit UUID validation
```
Comment (prcr) [inline README.md:214]: Fix the formatting for the new flag:

```suggestion
    * `--skip-commit-uuid-validation` [default: false] - Skip commit UUID validation
```

(I propose that we don't include the short flag name to make it consistent with all the others.

File: README.md [hunk 2]
```diff
@@ -211,6 +211,7 @@ Found [Clone] 7 duplicated lines with 10 tokens:
     * `--output` - Send the output results to a file
     * `--format` [default: text] - Change the output format (e.g. json)
     * `--commit-uuid` [default: latest commit of current git branch] - Set the commit UUID that will receive the results on Codacy
+    * `--skip-commit-uuid-validation` [default: false] - Skip commit UUID validation
```
Comment (prcr) [inline README.md:214]: Suggestion to clarify the purpose of the new flag:

```suggestion
    * `--skip-commit-uuid-validation` [default: false] - Force using a commit UUID even if it doesn't belong to the current git branch.
```

**Review by caxaria — CHANGES_REQUESTED** [review]
image

File: cli/src/main/scala/com/codacy/analysis/cli/command/AnalyseCommand.scala [hunk 3]
```diff
@@ -146,11 +146,12 @@ class AnalyseCommand(analyze: Analyze,
   }
 
   private def validateGitCommitUuid(repository: Repository,
```
[the remark this answers is not in the document]
Comment (heliocodacy) [reply cli/src/main/scala/com/codacy/analysis/cli/command/AnalyseCommand.scala:148]: Done

**Pull request merged**